draft-ietf-tls-tls13.md

```diff
@@ -2236,9 +2232,12 @@ ffdhe2048, etc.
 
 Values within "obsolete_RESERVED" ranges were used in previous versions
 of TLS and MUST NOT be offered or negotiated by TLS 1.3 implementations.
+The obsolete curves have various known or theoretical weaknesses and are
+no longer considered safe for general use. The supported values are more
+than sufficient for interoperability with existing and future servers.
 
```

Comment:
I don't think it's helpful, especially when we're talking about 571t.

I think it's fine to just say "not widely used".